.github/workflows/docs-redirects.yml

```
@@ -0,0 +1,24 @@
+name: docs-redirects
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
```

Restricting branches is unhelpful for testing against CI when needed:
```suggestion
on:
  push:
    paths:
      - "salt/docs/**"
      - "test/docs/redirects/**"
  pull_request:
    paths:
      - "salt/docs/**"
      - "test/docs/redirects/**"
    workflow_dispatch:

permissions:
  contents: read

concurrency:
  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
  cancel-in-progress: true
```